NitroxModel/DataStructures/NitroxInt3.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using NitroxModel.DataStructures.GameLogic;
+using ProtoBufNet;
+
+namespace NitroxModel.DataStructures
+{
+    /*
+     * NitroxInt3 model to allow NitroxModel to be decoupled from Assembly-csharp-firstpass
+     * as it is common between subnautica and below zero.
+     */
+    [Serializable]
+    [ProtoContract]
+    public class NitroxInt3
```

Comment:
What about making this a struct like the Sn version? 
![grafik](https://user-images.githubusercontent.com/23176718/96517076-8a3bb680-1268-11eb-8268-9cc33103d49a.png)

Reply:
Agreed- this should only be merged once it's not going to cause huge amounts of heap allocations to use it.